t04_refactored.py

```diff
@@ -183,8 +183,36 @@ def team_4_adv():
 
 
 def team_5_adv():
-    pass
-    # TODO Add your code here
+
+    #TEAM 5
+    dead = False
+    food = (input("choose what you want to eat today? [pasta/pizza/burger]")).lower()
+    if food == "pasta":
+        # good choice
+        print("you were lucky this time!")
+        print("now you get free pasta and get to live")
+    elif food == "pizza":
+        # bad choice
+        print("this is poisoned!")
+        print("you will experience death in 30 seconds!")
+        print("you didn't make it")
+        dead = True
+    elif food == "burger":
```

Comment:
There is no `else` condition which catches inputs that do not match any condition.